src/pages/Profile.css

```diff
@@ -64,3 +65,44 @@
 .btnBottle:hover {
     color: #c49e34;
 }
+
+.bottles-container{
+    padding: 2rem;
+    border: thick double #6D071A;
+    margin: 3rem;
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-auto-rows: 100px, auto;
+    gap: 0.3rem;
+}
+
+.descriptionBottles {
+    color: rgb(0, 0, 0);
+    font-size: 1.2rem;
+    text-decoration: underline goldenrod;
+    font-weight: 600;
+    border: solid 2px #000000;
+    background-color: whitesmoke;
+    padding: 1rem;
+    border-radius: 5%;
+    width: 50%;
+    height: 15vh;
```

Comment:
do you really need to fix width and height? try layout functionalities ;)

Reply:
width and height deleted